gettsim/transfers/kinderzuschl/kost_unterk.py

```diff
@@ -53,7 +53,7 @@ def kinderzuschlag_kaltmiete_m(
     return hh_id.replace(bruttokaltmiete_m_hh) * tax_unit_share
 
 
-def kinderzuschlag_heizkost_m(
+def kinderzuschl_heizkost_m(
```

Comment:
```suggestion
def kinderzuschl_heizkosten_m(
```
Never abbreviate in different ways (`heizkosten_m_hh`)